undertow/src/main/java/org/wildfly/extension/undertow/filters/ModClusterNodeDefinition.java

```diff
@@ -202,7 +203,7 @@ protected void handleNode(OperationContext context, ModClusterStatus.Node ctx, M
             @Override
             protected void handleNode(OperationContext context, ModClusterStatus.Node ctx, ModelNode operation) throws OperationFailedException {
                 final String domain = ctx.getDomain();
-                if(domain == null) {
+                if (domain == null) {
```

Comment:
Should I drop the whole formatting commit 419d96b and keep only the functional one d052e58?